packages/react/src/chain.ts

```diff
@@ -0,0 +1,7 @@
+import { IdentifierArray } from '@wallet-standard/base';
+
+type AssertSolanaChain<T> = T extends `solana:${string}` ? T : never;
+
+export type OnlySolanaChains<T extends IdentifierArray> = T extends IdentifierArray
+    ? AssertSolanaChain<T[number]>
+    : never;
```

Comment:
What does Pythnet use, for example? And what about "the rise of SVM rollups"? 😅

Reply:
I doubt Pythnet is even aware of this, since it's purely in the client. I imagine e.g. Eclipse should use either `solana:eclipse-mainnet` or `eclipse:mainnet`

Reply:
They can make `@pyth/react`, wrap this API, and override the types, maybe?